### PR TITLE
4658 DOD: Pa11y, npm upgrade, and integration tests

### PR DESCRIPTION
**Pull request by sutt0n**

### CODE REVIEW

File: web-client/pa11y/pa11y-petitionsclerk.js [hunk 1]
```diff
@@ -241,6 +241,18 @@ module.exports = [
     url:
       'http://localhost:1234/mock-login?token=petitionsclerk&path=/case-detail/103-19&info=correspondence-tab',
   },
+  {
+    actions: [
+      'wait for #tab-correspondence to be visible',
+      'click element #tab-correspondence',
+      'wait for #correspondence-documents-table to be visible',
+      'click element .edit-correspondence-button',
+      'wait for element .edit-correpsondence-header to be visible',
```
Comment (kkoskelin) [inline web-client/pa11y/pa11y-petitionsclerk.js]: classname is misspelled here -- is it misspelled elsewhere that also needs to be fixed?

File: web-client/src/views/Correspondence/EditCorrespondenceDocument.jsx [hunk 2]
```diff
@@ -60,7 +60,7 @@ export const EditCorrespondenceDocument = connect(
           <div className="grid-container padding-x-0">
             <div className="grid-row grid-gap">
               <div className="grid-col-12">
-                <h2 className="heading-1">
+                <h2 className="heading-1" id="edit-correpsondence-header">
```
Comment (kkoskelin) [inline web-client/src/views/Correspondence/EditCorrespondenceDocument.jsx]: ah here's a matching misspelling.  `correpsondence`

File: web-client/src/views/Correspondence/Correspondence.jsx [hunk 3]
```diff
@@ -70,9 +70,10 @@ export const Correspondence = connect(
                       {showAddCorrespondenceButton && (
                         <Button
                           link
-                          className="text-left padding-0 margin-left-1"
+                          className="text-left padding-0 margin-left-"
```
Comment (kkoskelin) [inline web-client/src/views/Correspondence/Correspondence.jsx]: is `margin-left-` a valid class name?

**Review by kkoskelin — CHANGES_REQUESTED** [review]
a few fixes needed, I think

File: web-client/src/views/Correspondence/Correspondence.jsx (hunk 3 above)
Comment (sutt0n) [reply web-client/src/views/Correspondence/Correspondence.jsx]: nope. fixed.

File: web-client/src/views/Correspondence/CorrespondenceHeader.jsx [hunk 4]
```diff
@@ -24,25 +24,28 @@ export const CorrespondenceHeader = connect(
               {showAddCorrespondenceButton && (
                 <Button
                   link
-                  aria-label="printable docket record"
+                  aria-label="add correspondence file"
```
Comment (kkoskelin) [inline web-client/src/views/Correspondence/CorrespondenceHeader.jsx:27]: thanks!

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**